Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM julia:1.9.0-bullseye
+
+ENV TERM=xterm 
+
+RUN apt-get clean && apt-get update && \
+apt-get install -y git
+
+RUN cd opt 
+
+WORKDIR /opt
+
+RUN git clone https://github.com/WilhelmusLab/ice-floe-tracker-pipeline.git && \ 
+git clone https://github.com/WilhelmusLab/IceFloeTracker.jl.git IceFloeTracker
```

Comment:
```suggestion
git clone https://github.com/WilhelmusLab/IceFloeTracker.jl.git
```

Reply:
This just gives the repo a simpler name in the environment, so users aren't thinking that `IceFloeTracker.jl` is a script. @mirestrepo pointed out that it looks like a script in the build commands, which is a bit confusing.

Reply:
Cool, didn't know that trick!